source/js/jquery-sortable.js

```diff
@@ -109,7 +110,7 @@
     // Ignore if element clicked is input, select or textarea
     onMousedown: function ($item, _super, event) {
       if (!event.target.nodeName.match(/^(input|select|textarea)$/i)) {
-        event.preventDefault()
+        if (event.type.match(/^mouse/)) event.preventDefault()
```

Comment:
If the event is a mouse event, it is still important to call `event.preventDefault()` to prevent text selection. Doing so does not interrupt click events if there is a click handler on the draggable object.

However, if the event is a touch event, we don't have to worry about text selection. We also *must not* call `event.preventDefault()`, since doing so will interrupt click events on the draggable object.